source/fresh-installation.rst

```diff
@@ -35,8 +35,8 @@ this were:
 
 .. code-block:: bash
 
-    sudo conda create -n collection-2018-2.1 -c nsls2-tag -y collection
-    sudo conda create -n analysis-2018-2.1 -c nsls2-tag -y analysis
+    sudo conda create -p collection-2018-2.1 -c nsls2-tag -y collection
```

Comment:
Still not quite there. `-n` (name) gets an environment name and `-p` (path) gets a full path. This and the block below need to be swapped.